docs/samples/Microsoft.ML.Samples/Dynamic/SDCACalibrated.cs

```diff
@@ -0,0 +1,68 @@
+        /// The type of those examples is <see cref="SamplesUtils.DatasetUtils.FloatLabelFloatFeatureVectorSample"/>,
+        /// which contains one bool label and a float feature vector.
+        /// </summary>
+        public static void SDCACalibrated_BinaryClassification()
+        {
+            // Generate C# objects as training examples.
+            var rawData = SamplesUtils.DatasetUtils.GenerateBinaryLabelFloatFeatureVectorSamples(100);
+
+            // Information in first example.
+            // Label: true
+            Console.WriteLine("First example's label is {0}", rawData.First().Label);
+            // Features is a 10-element float[]:
+            //   [0]	1.0173254	float
+            //   [1]	0.9680227	float
+            //   [2]	0.7581612	float
+            //   [3]	0.406033158	float
+            //   [4]	0.7588848	float
+            //   [5]	1.10602713	float
+            //   [6]	0.6421779	float
+            //   [7]	1.17754972	float
+            //   [8]	0.473704457	float
+            //   [9]	0.4919063	float
+            Console.WriteLine("First example's feature vector is {0}", rawData.First().Features);
+
+            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
+            // as a catalog of available operations and as the source of randomness.
+            var mlContext = new MLContext();
+
+            // Step 1: Read the data as an IDataView.
+            var data = mlContext.Data.ReadFromEnumerable(rawData);
+
+            // ML.NET doesn't cache data set by default. Caching is very helpful when working with iterative
+            // algorithms which needs many data passes. Since SDCA is the case, we cache.
+            data = mlContext.Data.Cache(data);
+
+            // Step 2: Create a binary classifier.
+            // We set the "Label" column as the label of the dataset, and the "Features" column as the features column.
+            var pipeline = mlContext.BinaryClassification.Trainers.StochasticDualCoordinateAscentCalibrated(labelColumn: "Label", featureColumn: "Features", l2Const: 0.001f);
+
+            // Step 3: Train the pipeline created.
+            var model = pipeline.Fit(data);
+
+            // Step 4: Make prediction and evaluate its quality (on training set).
+            var prediction = model.Transform(data);
+
+            var rawPrediction = mlContext.CreateEnumerable<SamplesUtils.DatasetUtils.CalibratedBinaryClassifierOutput>(prediction, false);
+
+            // Step 5: Inspect the prediction of the first example.
+            var first = rawPrediction.First();
+
+            Console.WriteLine("The first example actual label is {0}." +
+                " The trained model assigns it a score {1} and a probability of being positive class {2}.",
+                first.Label /*true*/, first.Score /*around 3.2*/, first.Probability /*around around 0.95*/);
```

Comment:
>around [](start = 93, length = 6)

one 'around' should suffice :)

Reply:
Couldn't agree more.

---
In reply to: [256258833](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256258833) [](ancestors = 256258833)